pubber/src/main/java/daq/pubber/PubberDeviceManager.java

```diff
@@ -22,23 +25,25 @@ public class PubberDeviceManager extends PubberManager implements DeviceManager
   private final PubberLocalnetManager localnetManager;
   private final PubberGatewayManager gatewayManager;
   private final PubberDiscoveryManager discoveryManager;
-  private Date lastConfigTimestamp;
+  private final List<SubBlockManager> subManagers;
 
   /**
    * Create a new instance.
    */
   public PubberDeviceManager(ManagerHost host, PubberConfiguration configuration) {
     super(host, configuration);
     systemManager = new PubberSystemManager(host, configuration);
-    pointsetManager = new PubberPointsetManager(host, configuration);
     localnetManager = new PubberLocalnetManager(host, configuration);
-    gatewayManager = new PubberGatewayManager(host, configuration);
+    pointsetManager = new PubberPointsetManager(host, configuration);
     discoveryManager = new PubberDiscoveryManager(host, configuration, this);
+    gatewayManager = new PubberGatewayManager(host, configuration);
+    subManagers = Arrays.asList(
+            systemManager, localnetManager, pointsetManager, discoveryManager, gatewayManager);
+    Collections.reverse(subManagers);
```

Comment:
Why reverse (why not just create it reversed initially?) -- also add a brief comment as to why the ordering is important.